src/adapters/postgres/cohortMembers-adapter.ts

```
@@ -901,6 +921,17 @@ export class PostgresCohortMembersService {
       INNER JOIN public."Roles" R
       ON R."roleId" = UR."roleId" ${whereCase}`;
 
+    // Debug logging for date filters
+    if (
+      whereCase.includes('CAST(CM."createdAt"') ||
+      whereCase.includes('CAST(CM."updatedAt"')
+    ) {
+      LoggerUtil.log(
+        `Generated query with date filter. WHERE clause: ${whereCase}`,
+        'COHORT_MEMBER_SEARCH'
```

ESLint

[error] Replace `'COHORT_MEMBER_SEARCH'` with `"COHORT_MEMBER_SEARCH"`

(prettier/prettier)